cdap-docs/admin-manual/source/hadoop-compatibility.rst

```diff
@@ -47,8 +47,8 @@ CDAP currently requires Internet access on the CDAP service nodes (or until the
 
 For a Distributed CDAP cluster, version |release|, you **must** install these Hadoop components:
 
-.. |cdh-version|       replace:: 5.0.0 through 5.7.0
-.. |cdh-version-hbase| replace:: 5.0.0 through 5.7.0 (1.2-cdh5.7.0)
+.. |cdh-version|       replace:: 5.1 through 5.8
+.. |cdh-version-hbase| replace:: 5.1 through 5.8 (1.2-cdh5.8.x)
```

Comment:
I think John is listing that as the HBase package version.  I think it's probably a bit overkill, and may introduce some confusion.  Since CDH version dictates the HBase version, it doesn't make sense to only list it for the top-end of the range.  If you are using CDH 5.3, your hbase package version is 0.98.6, which isn't reflected here at all.  I would say just remove it.